llm_studio/src/utils/config_utils.py

```diff
@@ -87,7 +75,7 @@ def _get_type_annotation_error(v: Any, type_annotation: Type) -> ValueError:
     )
 
 
-def convert_cfg_to_nested_dictionary(cfg: ConfigProblemBase) -> dict:
+def convert_cfg_to_nested_dictionary(cfg) -> dict:
```

Comment:
shall we still have `Any` type annotation there?

Reply:
Not sure, Any is rather generic type. 
I may have a look into better type annotations when using config classes, which is currently not possible due to cyclic imports.